receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -219,35 +231,147 @@ func TestTranslateV2(t *testing.T) {
 			request: writeV2RequestFixture,
 			expectedMetrics: func() pmetric.Metrics {
 				expected := pmetric.NewMetrics()
+				// For job "service-x/test", no explicit otel_scope is provided.
+				// fall back to BuildInfo defaults.
 				rm1 := expected.ResourceMetrics().AppendEmpty()
-				rmAttributes1 := rm1.Resource().Attributes()
-				rmAttributes1.PutStr("service.namespace", "service-x")
-				rmAttributes1.PutStr("service.name", "test")
-				rmAttributes1.PutStr("service.instance.id", "107cn001")
+				parseJobAndInstance(rm1.Resource().Attributes(), "service-x/test", "107cn001")
 				sm1 := rm1.ScopeMetrics().AppendEmpty()
+				sm1.Scope().SetName(defaultBuildName)
+				sm1.Scope().SetVersion(defaultBuildVersion)
+				// Expect 2 separate gauge metrics, one per timeseries.
 				sm1Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
 				sm1Attributes.PutStr("d", "e")
 				sm1Attributes.PutStr("foo", "bar")
-				// Since we don't check "scope_name" and "scope_version", we end up with duplicated scope metrics for repeated series.
-				// TODO: Properly handle scope metrics.
 				sm2Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
 				sm2Attributes.PutStr("d", "e")
 				sm2Attributes.PutStr("foo", "bar")
 
+				// For job "foo" with instance "bar", fallback to BuildInfo.
 				rm2 := expected.ResourceMetrics().AppendEmpty()
-				rmAttributes2 := rm2.Resource().Attributes()
-				rmAttributes2.PutStr("service.name", "foo")
-				rmAttributes2.PutStr("service.instance.id", "bar")
-				mAttributes2 := rm2.ScopeMetrics().AppendEmpty().Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
-				mAttributes2.PutStr("d", "e")
-				mAttributes2.PutStr("foo", "bar")
-
+				parseJobAndInstance(rm2.Resource().Attributes(), "foo", "bar")
+				sm2 := rm2.ScopeMetrics().AppendEmpty()
+				sm2.Scope().SetName(defaultBuildName)
+				sm2.Scope().SetVersion(defaultBuildVersion)
+				m3 := sm2.Metrics().AppendEmpty().SetEmptyGauge()
+				dp3 := m3.DataPoints().AppendEmpty()
+				dp3.Attributes().PutStr("d", "e")
+				dp3.Attributes().PutStr("foo", "bar")
+				return expected
+			}(),
+			expectedStats: remote.WriteResponseStats{},
+		},
+		{
+			name: "provided otel_scope_name and otel_scope_version",
```

Comment:
I'm not sure if I'm missing something, but I think this test case is covered by the test case `duplicated scope name and version`, isn't it? 

If that correct, could we remove this test case?